modules/ding_event/ding_event.module

```diff
@@ -344,6 +357,24 @@ function ding_event_form_views_exposed_form_alter(&$form, &$form_state) {
     if (empty($input)) {
       $form['reset']['#access'] = FALSE;
     }
+
+    if (!empty($form['#info']['filter-term_node_tid_depth'])) {
+      $label = $form['#info']['filter-term_node_tid_depth']['label'];
+      unset($form['#info']['filter-term_node_tid_depth']['label']);
+      $form['event_category_term_node_tid_depth']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
+
+    if (!empty($form['#info']['filter-og_group_ref_target_id_entityreference_filter'])) {
+      $label = $form['#info']['filter-og_group_ref_target_id_entityreference_filter']['label'];
+      unset($form['#info']['filter-og_group_ref_target_id_entityreference_filter']['label']);
+      $form['og_group_ref_target_id_entityreference_filter']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
+
+    if (!empty($form['#info']['filter-field_ding_event_target_tid'])) {
+      $label = $form['#info']['filter-field_ding_event_target_tid']['label'];
+      unset($form['#info']['filter-field_ding_event_target_tid']['label']);
+      $form['field_ding_event_target_tid']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-862

Reply:
This also seems to be default in main distro. What does this change?

Reply:
By default I assume it's already ajax-fied.  However, as in the example above this only seems to wrap the label/title in **h2** heading.

Reply:
And what purpose does that have?

Reply:
IDK.

Reply:
I see. There is not reason to change this to H2 - it can be styled.
This should not be ported.

Reply:
This applies for all three sections above. So it just rewrites the markup to aid styling by adding some classes. Reason for using h2 I think is that it's just one way of doing it. Whatever tag could be there with proper styles.

Reply:
![image](https://user-images.githubusercontent.com/574498/99979015-51cf5100-2daf-11eb-8236-2b0a5a4808c0.png)

Reply:
TBH I see no reason to keep this in the module - same can safely achieved in theme itself. I would suggest remove this too.

Reply:
I agree.